code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -734,3 +734,18 @@
 	taste_mult = 2
 	taste_description = "caramel"
 	reagent_state = SOLID
+
+/datum/reagent/consumable/char
+	name = "Char"
+	description = "Essence of the grill. Has strange properties when overdosed."
+	reagent_state = LIQUID
+	nutriment_factor = 5 * REAGENTS_METABOLISM
+	color = "#C8C8C8"
+	taste_mult = 6
+	taste_description = "smoke"
+	overdose_threshold = 25
+
+/datum/reagent/consumable/char/overdose_process(mob/living/carbon/M)
+	if(prob(10))
+		M.say(pick("I hate my wife.", "I just want to grill for God's sake.", "I wish I could just go on my lawnmower and cut the grass.", "Yep, Quake. That was a good game...", "Yeah, my PDA has wi-fi. A wife I hate."), forced = /datum/reagent/consumable/char)
```

Comment:
Maybe you should have an alternate set of lines for female (or gay, asexual, etc.) boomers?

Reply:
wow athath, you really think women cant have wives? so unprogressive of you...

Reply:
All true boomers are white middle aged fat dads that hate their wife

Reply:
boomer is a state of mind, not a gender ATHATH

Reply:
won't these messages fire pretty often

Reply:
also there is no message regarding pokemon creatures and pails of water

Reply:
i would say cringe but that's the goal here so i'm not going to give you that satisfaction